edward/models/random_variable.py

```diff
@@ -59,7 +59,8 @@ def __init__(self, *args, **kwargs):
     # need to temporarily pop value before __init__
     value = kwargs.pop('value', None)
     super(RandomVariable, self).__init__(*args, **kwargs)
-    self._kwargs['value'] = value  # reinsert (needed for copying)
+    if value is not None:
```

Comment:
If this doesn't happen, then the subsequent copies of the node made by `copy` don't have a `value` initializer, and you run into the same problem as before.

For instance, in the original use case, because Poisson doesn't have a random number generator in TensorFlow, I need to provide this `value=` initializer because `sample` isn't defined. If I don't stick `value` back into `kwargs`, there's no initializer when the Poisson node is copied, and it throws the same error.

Make sense?